samples/MvvmSample.Core/ViewModels/SamplePageViewModel.cs

```diff
@@ -34,7 +34,7 @@ public SamplePageViewModel()
         /// </summary>
         public IAsyncRelayCommand<string> LoadDocsCommand { get; }
 
-        private IReadOnlyDictionary<string, string> texts;
+        public IReadOnlyDictionary<string, string> Texts { get => texts; set => SetProperty(ref texts, value); }
```

Comment:
Nit: properties with explicit accessors should be formatted on new lines, one per accessor.